.github/workflows/velox_be.yml

```diff
@@ -560,3 +560,23 @@ jobs:
         if: ${{ always() }}
         run: |
           docker stop static-build-test-$GITHUB_RUN_ID || true
+  static-package-test:
```

Comment:
Maybe, better to name it like `build-script-test`. We can enrich the tests in this CI job in the future. Thanks!

Reply:
have done :)